graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/eacs/BEH7.kt

```diff
@@ -128,7 +70,7 @@ class BEH7 : GraphScenario() {
         }
     }
 
-    @Eac("When a directed graph has a sub-circle within a larger graph, return 2")
+    @Eac("When a given graph has a cycle and other connected nodes, return 2")
```

Comment:
🚫 I don't really understand what this is trying to say, so it needs rephrasing. Either way, saying "return 2" is most likely not right, because this seems to be talking about a local condition rather than a global condition -- the actual return value depends on what other cycles might be in the graph!